### PR TITLE
Added method 'withSound', updated example in readme

### PR DESCRIPTION
**Pull request by CharlieHorton**
Added method 'withSound' to define the notification sound, udpated the readme example to include the new method

### CODE REVIEW

**Review by HarmJan1990 — APPROVED** [review]
Just tested this and it works. Please add.

File: src/Services/Larafirebase.php [hunk 1]
```diff
@@ -101,6 +110,7 @@ public function sendNotification($tokens)
                 'body' => $this->body,
                 'image' => $this->image,
                 'icon' => $this->icon,
+                'sound' => $this->sound,
```
Comment (gentritabazi01) [inline src/Services/Larafirebase.php:113]: You should add this to `sendNotification` & `sendMessage`.

**Comment by gentritabazi01** [comment]
Title of PR should be:

`[1.x] Add method withSound`

**Comment by kaleemshoukat** [comment]
withSound() is not working in laravel 9. I had to customize the package as per your commit. Please recheck it.

**Comment by gentritabazi01** [comment]
Old PR, also duplicate of #45

**Pull request closed without merging**